posthog/models/cohort/cohort.py

```diff
@@ -215,7 +215,8 @@ def calculate_people(self, new_version: int, batch_size=10000, pg_batch_size=100
 
                 cursor += batch_size
                 persons = self._clickhouse_persons_query(batch_size=batch_size, offset=cursor)
-                time.sleep(5)
```

Comment:
i am just wondering why do we sleep here? 😮

Reply:
Totally fair question. @hazzadous Do you think we can get rid of this now? https://github.com/PostHog/posthog/pull/8598

Reply:
It’s to avoid hammering the database. There might be other ways but this is simple.

Alternative to this change you could just patch sleep in tests